dronekit/__init__.py

```diff
@@ -1845,6 +2018,12 @@ def initialize(self, rate=4, heartbeat_timeout=30):
             self._master.mav.request_data_stream_send(0, 0, mavutil.mavlink.MAV_DATA_STREAM_ALL,
                                                       rate, 1)
 
+        #Request an AUTOPILOT_VERSION packet
+        def send_capa_request(vehicle, name, m):
```

Comment:
I prefer full descriptive names. We're in Python. `send_capabilities_request` or even better `request_capabilities`. Does this need to be public or can it be hidden?